diff --git a/cloud/remote-separation/worker/Dockerfile b/cloud/remote-separation/worker/Dockerfile
index a10e90a..6794cc4 100644
--- a/cloud/remote-separation/worker/Dockerfile
+++ b/cloud/remote-separation/worker/Dockerfile
@@ -1,4 +1,4 @@
-ARG PYTHON_BASE=python:3.12.14-slim-bookworm@sha256:782412e85d0f0984994c290652577d4018aff08145c85b262bb63dc0c7522254
+ARG PYTHON_BASE=python:3.12.14-slim-trixie@sha256:2f17fc044b579bab302c2e8054d3a686e2cb9a83de48e70534b94cd8ebbe06a9
 
 FROM ${PYTHON_BASE} AS builder
 
